Core/Core/Inbox/ComposeMessage/ViewModel/ComposeMessageViewModel.swift

```diff
@@ -20,15 +20,18 @@ import Combine
 import CombineExt
 import CombineSchedulers
 
-class ComposeMessageViewModel: ObservableObject {
+public protocol ComposeMessageDelete: AnyObject {
+    func didSendMailSuccessfully()
+}
```

Comment:
Since there's only one method/event here I wonder if we could do this via passing a combine subject. That would be more in-line with our current approach.